### ISSUE TITLE
Minimize window after unlocking database sub option

### ISSUE THREAD

**Issue by sebzag** [keepassxreboot/keepassxc#8873]
It would be convenient to add a sub option to the feature _Minimize window after unlocking database_ to not occur when the user opens the window manually.

**Comment by droidmonkey** [comment]
That is precisely when the user wants it to be minimized. I don't understand this use case.

**Comment by sebzag** [comment]
Actually quite the contrary, you don't want it when you manually open the window because that's when you want to be using the program. This option is only beneficial for automated use cases outside of the program i.e. logging on through a browser.

**Comment by droidmonkey** [comment]
When you log in from browser, etc it will return the window to the original state (minimized or in tray) and may not even show the window on Windows and Linux. So no, I don't agree... users who use this setting specifically have it set for when they manually open and unlock the program.

**Comment by phazzzy** [comment]
Just looked for an option like that and couldn't find it.

@droidmonkey, there are at least two custom scripts 

1) I work in a browser and use an extension to automatically fill in passwords. In this case I only need to unlock the base to access them. A popup window appears, gets in the way, annoys me, and I enable automatic hiding.

2) I need to get a password from a service where there is no autofill. For example, not in a browser. In this case I need to open the password database and find the password myself. After entering the master password, according to the settings, the window also automatically disappears, and you have to open it manually. 

Bottom line, in both scenarios there is an inconvenient point. Different contexts of use require different behavior. One setting does not solve the two problems because they are mutually exclusive.

Would be convenient if I can separately adjust the behavior of the window when you open the base through your browser extension, and when you open keepassxc via a shortcut or an icon in the tray.

**Comment by droidmonkey** [comment]
For the second one you should be using Auto-Type 

> In this case I only need to unlock the base to access them. A popup window appears, gets in the way, annoys me, and I enable automatic hiding.

I don't understand what you are saying here. Do you not want to unlock your database? Are you confused about the prompts? Do you have your lock timeout set too short so you have to unlock too frequently?

**Comment by phazzzy** [comment]
Unlocking the database ≠ opening a window.
But opening the window = unlocking the database. Except opening the password generator.

What confuses me is that in one scenario I need to access the keepassxc window to find some desired data or change settings. At the same time, I don't need autofill or automatic hiding of the window after entering the master password. I just want to open the program window for some purposes of my own.

And in the other case, which is more often, I need to allow an extension access to the database to use autofill. In this case, I don't need to see the keepassxc window. Unless it's just for entering the master password and then going back to the browser.


A click in the tray should always show the window. It does not need to be hidden automatically after entering the master password.
Unlocking the database through a browser extension, should always hide the window.
One of the scenarios is not possible with the available behavior settings.

There is also another problem, to open the password generator you don't need to enter the master password. Now the extension requires unlocking the database and entering a password to open it

**Comment by phazzzy** [comment]
#5710 Relevant to what I want, but it's better to set the behavior once and forget it, instead of having to face each time you enter the master password

**Comment by droidmonkey** [comment]
Unfortunately I just do not understand how this requested setting would be implemented. I much prefer the modifier key to bypass minimizing as it is a deliberate user action when desired.
